cli/test-utils/src/util.rs

```diff
@@ -588,6 +588,11 @@ impl TestCommand {
         self.expect_success(output)
     }
 
+    /// Runs the command and asserts that it resulted in success
+    pub fn assert_success(&mut self) {
+        self.output();
+    }
```

Comment:
why do we need this? just use `output` or rename `output()`?

Reply:
we also have `assert_err`, I thought this is more descriptive then just calling `.output`